cclient/serialization/json/json_serializer.c

```diff
@@ -49,6 +49,44 @@ retcode_t utarray_to_json_array(UT_array* ut, cJSON* json_root,
   return RC_OK;
 }
 
+retcode_t json_array_to_int_array(cJSON* obj, const char* obj_name,
+                                  int_array* in) {
+  cJSON* json_item = cJSON_GetObjectItemCaseSensitive(obj, obj_name);
+  if (cJSON_IsArray(json_item)) {
+    cJSON* current_obj = NULL;
+    int_array_allocate(in, cJSON_GetArraySize(json_item));
+    int* in_arr_p = in->array;
+
+    cJSON_ArrayForEach(current_obj, json_item) {
+      if (current_obj->valuestring != NULL) {
+        *in_arr_p = atoi(current_obj->valuestring);
```

Comment:
what if the value of string bigger than (2^31)-1 on a 32bit platform?

Reply:
could I use the int* in int_array to point to a int array whose each element is each the char in the balances string?